notebooks/priority_fields/get_cred.ipynb

```diff
@@ -0,0 +1,199 @@
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "prk6kZuq6jQH"
+      },
+      "source": [
+        "def similar(target_phrases, file):\n",
+        "    \"\"\"GET RID OF PUNCT\"\"\"\n",
+        "    matcher = Matcher(nlp.vocab)\n",
+        "    matcher.add('target_phrases', target_phrases)\n",
+        "    \n",
+        "    return matcher(file, as_spans=True)"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "FwF5VUzEt_jm"
+      },
+      "source": [
+        "def get_credibility(self):  \n",
+        "    \"\"\"\n",
+        "    Returns whether or not the Respondent was identified as credible by their\n",
+        "    assigned judge / court. \n",
+        "    \"\"\" \n",
+        "    pattern = [\n",
+        "                # most important phrasing \n",
+        "                [{\"LOWER\": \"court\"}, {\"LOWER\": \"finds\"}, {\"LOWER\": \"respondent\"}, {\"LOWER\": \"generally\"}, {\"LOWER\": \"credible\"}],\n",
+        "                [{\"LOWER\": \"court\"}, {\"LOWER\": \"finds\"}, {\"LOWER\": \"respondent\"}, {\"LOWER\": \"testimony\"}, {\"LOWER\": \"credible\"}],\n",
+        "                [{\"LOWER\": \"court\"}, {\"LOWER\": \"finds\"}, {\"LOWER\": \"respondent\"}, {\"LOWER\": \"credible\"},\n",
+        "\n",
+        "                # next in omportance\n",
+        "                [{\"LOWER\": \"credible\"}, {\"LOWER\": \"witness\"}],\n",
+        "                [{\"LOWER\": \"generally\"}, {\"LOWER\": \"consistent\"}],\n",
+        "                [{\"LOWER\": \"internally\"}, {\"LOWER\": \"consistent\"}],\n",
+        "                [{\"LOWER\": \"sufficiently\"}, {\"LOWER\": \"consistent\"}],\n",
+        "                [{\"LOWER\": \"testified\"}, {\"LOWER\": \"credibly\"}],\n",
+        "                [{\"LOWER\": \"testimony\"}, {\"LOWER\": \"credible\"}],\n",
+        "                [{\"LOWER\": \"testimony\"}, {\"LOWER\": \"consistent\"}],\n",
+        "\n",
+        "                # least importance \n",
+        "                [{\"LOWER\": \"coherent\"}],\n",
+        "                [{\"LOWER\": \"plausible\"}]\n",
+        "              ]\n",
+        "\n",
+        "    similar_cred = similar(target_phrases=pattern, file=self.doc)\n",
+        "    \n",
+        "    if similar_cred:\n",
+        "        # following code adds matches to a list, may or may not be needed in\n",
+        "        # the front end.\n",
+        "        \n",
+        "        # cred = []\n",
+        "        # for phrase in similar_cred:\n",
+        "        #     if phrase.text.lower() not in cred:\n",
+        "        #         cred.append(phrase.text.lower())\n",
+        "\n",
+        "        return 'Respondent was found credible' # this could be changed to bool\n",
+        "\n",
+        "    return 'Respondent was not found credible' # same for this"
```

Comment:
I would probably have this return a statement that isn't this strong, but the client may think otherwise.